content/en/blog/2025/otel-day.md

```diff
@@ -0,0 +1,196 @@
+---
+title:
+  'OTel Community Day + Open Observability Summit: One Big Day for Open Source
+  Observability'
+linkTitle: OTel Community Day
+date: 2025-06-13
+author: '[Isabella Langan](https://github.com/isabella-rose-l) (Causely)'
+# prettier-ignore
+cspell:ignore: Langan Causely Horovits Alok Bhide Radlein Kuncham Canuel Danielson OllyGarden Vashistha Kumar Baylis Roshore Budhaditya Bhattacharya Ravindra Bhargava Distrogen Braydon Kains Mansi Agrawal Todea Aircall Liudmila Molkova Cijo Jernigan Spec-tacular Helmuth Krietz Prasad Mujumdar Okahu Vijay Heffner Huxing Zhang Minghui
+---
+
+On **June 26, 2025**,
+[OTel Community Day and Open Observability Summit](https://events.linuxfoundation.org/open-observability-summit-otel-community-day/)
+are joining forces at
+[Open Source Summit North America](https://events.linuxfoundation.org/open-source-summit-north-america/).
+The OpenTelemetry governance and technical committees, along with project
+maintainers, are excited to present two community-led tracks, one co-located
+event, and a shared goal: bring the observability community together to
+collaborate, learn, and shape the future of open source observability.
+
+**Open Observability Summit** is for developers, operators, and technical
+leaders who rely on open source to gain deeper insight into their systems. Every
+session is grounded in real-world experience and shaped by the community. Expect
+hands-on learning, practical talks, and open dialogue.
+
+**OTel Community Day** is a space for the OpenTelemetry community to celebrate
+wins, share knowledge, and connect over where observability is headed. Our
+[contributor experience survey results](../contribex-survey-results/)
+show that OTel contributors are excited to collaborate in person, where they can
+not only learn from each other and but also help end users with practical
+insights and real-world examples. Whether you're building OTel or building with
+it, you'll find your people here.
+
+## Schedule
+
+For convenience, a snapshot of the schedule is included below. Open it to search
+for the topics of interest.
+
+<details>
+<summary class="mb-3">View schedule</summary>
+
+### Welcome + Keynotes
+
+- **[Welcome + Opening Remarks](https://otelopenobservabilityna25.sched.com/event/223Or/welcome-+-opening-remarks-austin-parker-honeycomb)**<br>
+  by Austin Parker, honeycomb.io<br> 9:00am – 9:05am<br> Bluebird Ballroom 1C
+  (Level 1 - Terrace Level)
+
+- **[Keynote: Hybrid Cloud Architecture: Making Big Bets on Open Standards](https://otelopenobservabilityna25.sched.com/event/24dNu/keynote-hybrid-cloud-architecture-making-big-bets-on-open-standards-margaret-dawson-chronosphere)**<br>
+  by Margaret Dawson, Chronosphere<br> 9:08am – 9:18am<br> Bluebird Ballroom 1C
+  (Level 1 - Terrace Level)
```

Comment:
Here and elsewhere, drop the path part after the event ID, it is useless (go ahead and try any other path ending and you'll see):

```suggestion
- **[Keynote: Hybrid Cloud Architecture: Making Big Bets on Open Standards](https://otelopenobservabilityna25.sched.com/event/24dNu/)**<br>
  by Margaret Dawson, Chronosphere. 9:08am – 9:18am
```